.github/workflows/eslint.yaml

```diff
@@ -0,0 +1,18 @@
+name: ESLint
```

Comment:
oh my god a github action, what is this a job?

Reply:
An experiment, because I'm addicted to linters. Especially when I don't know the language I'm programming in.